.builds/freebsd.yml

```diff
@@ -25,5 +25,5 @@ tasks:
       cargo clippy --all-targets
   - oldstable: |
       cd copypasta
-      rustup toolchain install --profile minimal 1.65.0
-      cargo +1.65.0 test
+      rustup toolchain install --profile minimal 1.66.0
+      cargo +1.66.0 test
```

Comment:
MSRV bumps should be split into a separate commit (before the actual change obviously).

Reply:
Thanks, done!